common/include/x86emitter/legacy_instructions.h

```diff
@@ -15,1349 +15,251 @@
 
 #pragma once
 
+#if defined(__linux__) && defined(__clang__)
+#define ATTR_DEP [[deprecated]]
+#define ATTR_DEP [[deprecated]]
```

Comment:
Initially I used several define to separate MXX/SSE (avoid severals thousand of warning at once). I badly deleted the line. I saw it but I wasn't in the mood to rebase everything.